jailer.go

```diff
@@ -367,6 +374,24 @@ func LinkFilesHandler(rootfs, kernelImageFileName string) Handler {
 			}
 
 			m.cfg.KernelImagePath = kernelImageFileName
+
+			for _, fifoPath := range []string{m.cfg.LogFifo, m.cfg.MetricsFifo} {
+				if fifoPath == "" {
+					continue
+				}
+				fileName := filepath.Base(fifoPath)
+				if err := linkFileToRootFS(
+					m.cfg.JailerCfg,
+					filepath.Join(rootfs, fileName),
+					fifoPath,
+				); err != nil {
+					return err
+				}
+				if err := os.Chown(filepath.Join(rootfs, fileName), *m.cfg.JailerCfg.UID, *m.cfg.JailerCfg.GID); err != nil {
+					return err
+				}
```

Comment:
There needs to be an update to the `m.cfg.LogFifo` and `m.cfg.MetricsFifo` path to point to the root directory.

```
*fifoPath = fileName
```